src/App.js

```diff
@@ -35,6 +35,7 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+                data?.Workloads ? dispatch(setWorkloads(data.Workloads)) : dispatch(setWorkloads(undefined));
```

Comment:
Can be simplified with
```suggestion
                dispatch(setWorkloads(data?.Workloads))
```

Reply:
oh slick yeah i'll change that